xl_auth/commands.py

```diff
@@ -95,6 +97,21 @@ def clean():
 
 
 @click.command()
+@click.option('--email', required=True, default=None, help='Email for user')
+@click.option('--full-name', default=None, help='Full name for user (default: None)')
+@click.option('--password', default='password', help='Password for user (default: password)')
+@click.option('--active', default=True, is_flag=True, help='Activate account (default: True)')
+@click.option('--is-admin', default=False, is_flag=True, help='Create admin user (default: False)')
+@with_appcontext
+def create_user(email, full_name, password, active, is_admin):
+    """Create user account."""
+    full_name = full_name or email
+    user = User.create(email=email, full_name=full_name, password=password,
+                       active=active, is_admin=is_admin)
+    click.echo('Created account with login {0}'.format(user.email))
```

Comment:
Suggestion: For people who doesn't read the help text defaults for password and just followed the README quickstart steps, how about printing out

    Created account with login <email>:<password>

?